qiskit_nature/problems/sampling/protein_folding/bead_contacts/contact_map.py

```diff
@@ -0,0 +1,71 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""A class that stores contacts between beads of a peptide as qubit operators."""
+from typing import Dict
+
+from .contact_map_builder import (
+    _create_contact_qubits,
+)
+from ..peptide.peptide import Peptide
+
+
+class ContactMap:
```

Comment:
Just to use this as an example: currently, the only documentation which one can navigate to are those classes which are listed in `qiskit_nature.problems.sampling.protein_folding`. Anything else appears to not be accessible.
This is fine, if the user never needs to create e.g. a `ContactMap` manually. Otherwise, we should address this and make sure that all public classes actually get included into the docs.

Reply:
For now I did expose all the classes that I intended to... Perhaps as we add more functionalities and maybe allow more freedom for the user to define things manually then we will expose more.

Reply:
I just wanted to make sure that e.g. the tutorial will not call for the user to write a `ContactMap` themselves and it not being visible in the docs. If (for now at least) the `ContactMap` is only used internally that is fine :+1: